mycpp/gc_dict.h

```diff
@@ -46,7 +52,12 @@ class Dict {
   }
 
   Dict(std::initializer_list<K> keys, std::initializer_list<V> values)
-      : GC_CLASS_FIXED(header_, field_mask(), sizeof(Dict)) {
+      : GC_CLASS_FIXED(header_, field_mask(), sizeof(Dict)),
+        len_(),
+        capacity_(),
+        entry_(),
+        keys_(),
+        values_() {
     assert(len_ == 0);
     assert(capacity_ == 0);
     assert(entry_ == nullptr);
```

Comment:
Please remove all the assert() and DCHECK() as well, since they're now redundant

Personally I would be explicit with `len_(0)` and `keys_(nullptr)`, since I  didn't know you could leave them out !  :-)